jitsi_meet_wrapper/README.md

```diff
@@ -159,6 +159,19 @@ android {
 }
 ```
 
+If you still want to compile your app for a SDK version lower than 23, you can add the following line to your `AndroidManifest.xml`:
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          xmlns:tools="http://schemas.android.com/tools"
+          package="dev.saibotma.jitsi_meet_wrapper_example">
+
+    <!-- Add this the following line -->
+    <uses-sdk tools:overrideLibrary="dev.saibotma.jitsi_meet_wrapper, org.jitsi.meet.sdk, co.apptailor.googlesignin, com.calendarevents, com.reactnativecommunity.asyncstorage, com.reactnativecommunity.netinfo, com.rnimmersive, com.corbt.keepawake, com.BV.LinearGradient, com.horcrux.svg, com.oney.WebRTCModule, com.ocetnik.timer, com.kevinresol.react_native_default_preference, com.learnium.RNDeviceInfo, com.amplitude.reactnative, com.reactnativegooglesignin, com.reactnativecommunity.clipboard, com.swmansion.gesturehandler.react, org.linusu, org.reactnative.maskedview, com.reactnativepagerview, com.oblador.performance, com.swmansion.reanimated, com.th3rdwave.safeareacontext, com.swmansion.rnscreens, com.reactnativecommunity.slider, org.devio.rn.splashscreen, com.brentvatne.react, com.reactnativecommunity.webview"/>
```

Comment:
If there is any source where you got all the package names from, then it would be great if you could link the source in a comment. If not, then please ignore this comment.

Reply:
I got these package names by setting the `minSdkVersion` to < 23 and then start an Android build with `flutter run` (it doesn't matter which SDK version the device has). You will then get this error message:
```
/Users/nils/Desktop/Projects/sharezone/sharezone-app/app/android/app/src/main/AndroidManifest.xml:26:5-804 Error:
	uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library [org.jitsi.react:jitsi-meet-sdk:5.0.2] /Users/nils/.gradle/caches/transforms-2/files-2.1/223c66d362c97ac58a06ece91d65f1c3/jetified-jitsi-meet-sdk-5.0.2/AndroidManifest.xml as the library might be using APIs not available in 21
	Suggestion: use a compatible library with a minSdk of at most 21,
		or increase this project's minSdk version to at least 23,
		or use tools:overrideLibrary="org.jitsi.meet.sdk" to force usage (may lead to runtime failures)

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processProdReleaseManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library [org.jitsi.react:jitsi-meet-sdk:5.0.2] /Users/nils/.gradle/caches/transforms-2/files-2.1/223c66d362c97ac58a06ece91d65f1c3/jetified-jitsi-meet-sdk-5.0.2/AndroidManifest.xml as the library might be using APIs not available in 21
  	Suggestion: use a compatible library with a minSdk of at most 21,
  		or increase this project's minSdk version to at least 23,
  		or use tools:overrideLibrary="org.jitsi.meet.sdk" to force usage (may lead to runtime failures)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 23s
```

I added the package name from the error message and started the Android build again. Until the Android build was successful.

You can see in https://github.com/SharezoneApp/sharezone-app/pull/164 the package names which I needed to add when migrating from the `jitsi_meet` package to this package.

Reply:
Okay, makes sense. I think we should add this explanation to README.md because otherwise we'd have to keep the list of packages up to date in case a new version of Jitsi requires additional overrides. 

What do you think?

Reply:
Hmm, yes this could be one option, but I think keeping the list up to date is a better option because it took me like half an hour to get the package names (start build, wait 2-3 min, add package, start again). We could add a hint that you can open a PR if the list outdated?

Reply:
I have added a short explanation and also linked https://github.com/saibotma/jitsi_meet_wrapper/pull/32/files#r867651938.